xdem/demcollection.py

```diff
@@ -47,8 +47,7 @@ def __init__(self, dems: Union[list[gu.georaster.Raster], list[xdem.DEM]],
 
         # Find the sort indices from the timestamps
         indices = np.argsort(self.timestamps.astype("int64"))
-        self.dems = np.empty(len(dems), dtype=object)
-        self.dems[:] = [dems[i] for i in indices]
+        self.dems = [dems[i] for i in indices]
```

Comment:
Are you sure both are equivalent? Before `self.dems` was a `np.ndarray`, but now it's not.

Reply:
Yes I think so (all tests passing so the `demcollection.py` module seems to be functioning correctly). 
Before, `self.dems` was an `np.array` just used for iterations through the `xdem.DEM` as `object` types. By using `[:]` on previous line 51, it broadcasted them differently with the `__array_interface__` within the list. Now it's consistent (and understandable). I guess we'll fix it later if we find an issue (and make the tests more complete).

Reply:
👍